### ISSUE TITLE
Implement a Banlist for ODs

### ISSUE THREAD

**Issue by MCOfficer** [MCOfficer/odcrawler-discovery#96]
We had an incident with an OD supposedly containing CP. We don't want to import it if it ever finds out way into our scan folders again, so a banlist would be useful.

**Comment by Chaphasilor** [comment]
Would also be useful to block known honeypots like indexof.ir from ending up in the dabase :)

**Comment by upintheairsheep** [comment]
No, we need a banlist for common but gigantic OD sites like phonetones for instance.
